app/(app)/session/[session].tsx

```diff
@@ -1,40 +1,184 @@
+                  </StyledText>
+                )}
+              </StyledText>
+            )}
+
+            <Space size={16} />
+
+            <Image
+              source={{ uri: session?.session_image || '' }}
+              style={styles.image}
+              contentFit="fill"
+              contentPosition="left center"
+            />
+          </View>
+
+          <View style={[styles.centered, { borderColor: dark ? colors.background : colors.border }]}>
+            <StyledText font="light">{getSessionTimesAndLocation(session?.slug || '')}</StyledText>
+
+            <Space size={16} />
+
+            <View style={[styles.chip, { backgroundColor: dark ? colors.background : colors.text }]}>
+              <StyledText style={[styles.chipText, { color: dark ? colors.text : colors.background }]}>
+                #{session?.session_level}
+              </StyledText>
+            </View>
+          </View>
+
+          <View style={styles.withPadding}>
+            {session?.speakers && session?.speakers.length > 1 ? (
+              <View>
+                <StyledText font="medium" size="lg">
+                  Twitter Handles
+                </StyledText>
+
+                <Space size={16} />
+
+                <Row>
+                  {session?.speakers.map((speaker, index) => (
+                    <Pressable
+                      key={index.toString()}
+                      style={[styles.button, { borderColor: colors.primary, backgroundColor: colors.background }]}
+                    >
+                      <FontAwesome5 name="twitter" size={20} color={colors.primary} />
+                      <Space size={4} horizontal />
+                      <StyledText font="medium" style={{ color: colors.primary }}>
+                        {speaker.twitter ? getTwitterHandle(speaker.twitter) : 'N/A'}
+                      </StyledText>
+                    </Pressable>
+                  ))}
+                </Row>
+              </View>
+            ) : (
+              <Row>
+                <StyledText font="medium" size="lg">
+                  Twitter Handle
+                </StyledText>
+
+                <Pressable style={[styles.button, { borderColor: colors.primary, backgroundColor: colors.background }]}>
+                  <FontAwesome5 name="twitter" size={20} color={colors.primary} />
+                  <Space size={4} horizontal />
+                  <StyledText font="medium" style={{ color: colors.primary }}>
+                    {session?.speakers[0]?.twitter ? getTwitterHandle(session?.speakers[0]?.twitter) : 'N/A'}
+                  </StyledText>
+                </Pressable>
+              </Row>
+            )}
+          </View>
+        </View>
+      </MainContainer>
+
+      {/** Floating action button. TODO: Add share functionality */}
+      <Pressable style={[styles.fab, { backgroundColor: colors.tertiary }]}>
+        <MaterialCommunityIcons name="share" size={30} color="white" />
+      </Pressable>
```

Comment:
We should add bottom margin to the screen to allow scrolling the twitter handles past the floating button. Since both of them are Pressables, they should not overlap with each other.

<img width="576" alt="image" src="https://github.com/droidconKE/droidconKE2023ReactNative/assets/5578688/056759ff-5f1a-4f8c-95fb-36411485b672">

Reply:
done. added `marginBottom` on `main`, and space after the twitter handles